Library/Tracking/KSRAnalytics.swift

```diff
@@ -836,14 +843,16 @@ public final class KSRAnalytics {
   public func trackProjectCardClicked(page: PageContext,
                                       project: Project,
                                       checkoutData: CheckoutPropertiesData? = nil,
+                                      typeContext: TypeContext? = nil,
```

Comment:
I wonder if the default makes more sense to be `project` since that is the argument we are using in every situation except results

Reply:
I'm not sure about that because it's not specific to results, I have seen one other ticket that has a different `typeContext`  and there could be more. Most of the ones that has projects probably have a ticket that changes the `typeContext` to something else.

https://kickstarter.atlassian.net/jira/software/projects/EP/boards/53?label=ios%2Cphase_2&selectedIssue=EP-431